dss/__init__.py

```diff
@@ -68,12 +69,12 @@ def common_error_handler(exception):
             problem['status'] = exception.code
             problem['code'] = exception.__class__.__name__
             problem['title'] = exception.description
-        return ConnexionResponse(
+        return FlaskApi.get_response(ConnexionResponse(
```

Comment:
I think `error.py` doesn't need it because it's deeper in the middleware stack, and has a function above it that will translate its ConnexionResponse to Flask format.

`common_error_handler` on the other hand sits outside the connexion middleware stack, immediately on the Flask stack, it seems.